test/runtests.jl

```diff
@@ -4,7 +4,9 @@ using ManifoldsBase
 
     num_ambiguities = length(Test.detect_ambiguities(ManifoldsBase))
```

Comment:
In Manifolds even the call here failed, so would it maybe be better to even put this line into the version test?

Reply:
Are you sure? Just calculating the number of ambiguities shouldn't fail.

Reply:
Well, it was with `Test.detect_ambiguities(Base)` which ran into a Stack Overflow, so in `Manifolds` also that call is within the `if`.

Reply:
But it's also not that bad, so do what you feel is better; I'll already accept this :)

Reply:
OK, I've put it in that `if`.